docs/Exclude_Object.md

```diff
@@ -0,0 +1,106 @@
+# Exclude Obects
+
+The `[exclude_object]` module allows Klipper to exclude objects while a print is in progress.
+To enable this feature include an [exclude_object config section](Config_Reference.md#exclude_object)
+(also see the [command reference](G-Codes.md#exclude-object).)
+
+Unlike other 3D printer firmware options, a printer running Klipper utilizes a suite of
+components and users have many options to choose from.  Therefore, in order to provide a
+a consistent user experience, the `[exclude_object]` moudle will establish a contract or API
+of sorts.  The contract covers the contents of the gcode file, how the internal state of the
+module is controlled, and how that state is provided to clients.
+
+## Workflow Overview
+A typical worfklow for printing a file might look like this:
+1. Slicing is completed and the file is uploaded for printing.  During the upload, the file
+   is processed and `[exclude_object]` markup is added to the file.
+1. When printing starts, Klipper will reset the `[exclude_object]` status.
```

Comment:
It would be preferable to expand the numbering (eg, 1, 2, 3) so the document is readable via text files.